OpenRA.Mods.Common/Traits/World/MapCreeps.cs

```diff
@@ -15,22 +15,36 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Controls the 'Creeps' checkbox in the lobby options.")]
-	public class MapCreepsInfo : TraitInfo<MapCreeps>
+	public class MapCreepsInfo : ITraitInfo, ILobbyOptions
 	{
 		[Desc("Default value of the creeps checkbox in the lobby.")]
 		public readonly bool Enabled = true;
 
 		[Desc("Prevent the creeps state from being changed in the lobby.")]
 		public readonly bool Locked = false;
+
+		IEnumerable<LobbyOption> ILobbyOptions.LobbyOptions(Ruleset rules)
+		{
+			yield return new LobbyBooleanOption("creeps", "Worms", Enabled, Locked);
```

Comment:
How about exposing "Worms" to the trait info so this becomes configurable and more general?